test/classes/initializers/postInit/throwing-superclass3.good

```diff
@@ -0,0 +1,3 @@
+throwing-superclass3.chpl:19: error: conflicting throws for 'Bar.postinit()'
+throwing-superclass3.chpl:7: error: throwing function 'Foo.postinit()'
+throwing-superclass3.chpl:19: error: overridden by non-throwing function 'Bar.postinit()'
```

Comment:
Is this really strictly correct?  Could `Bar.postinit()` have an explicit call to `super.postinit()`, which it catches and handles?  (Not sure what that would mean?) But then `Bar.postinit()` wouldn't need to be marked throws.

Relatedly and/or unrelatedly depending on the first answer, I don't think it's a matter of matching for overriding like the error here says.  The postinit calls aren't called by virtual dispatch, right?  The `new Bar()` expression has the real type of the object (in this test and in general) and would be invoking `.postinit()` on an object it knows to be a `Bar`, right, so it doesn't need to match throwingness for override reasons.  (Supporting evidence: the `Bar.postinit()` definition doesn't have the `override` keyword.)

Reply:
@cassella : I think you're right that this error is inappropriate and that postinit()s shouldn't be treated like they were overrides.  Yesterday, I was a little surprised by it, but considered it to be a pre-existing condition, so didn't really think about it much and just snapshotted it for this test to lock in the cases that I'd enabled.  But on reflection, since we couldn't even declare a postinit as `throws` before, it arguably isn't a pre-existing condition so much as a latent one, so is something I should try to handle here.  I'll take a look at it.

Reply:
58e5b6d makes this behave a more reasonable way I think.  The new error is a bit obscure in that it relies on understanding that the compiler has inserted an implicit call to `super.postinit()` in the subclass `postinit()` but I think it's accurate and strictly better than last night's version.